### PR TITLE
adding an example for dynamodb

### PR DESCRIPTION
**Pull request by trish11953**
Just an example to update the docs!
 Let me know what you think.

----
:books: Documentation preview :books:: https://griptape--45.org.readthedocs.build/en/45/

### CODE REVIEW

File: docs/griptape-framework/data/vector-stores.md [hunk 1]
```diff
@@ -152,4 +152,62 @@ The following methods are available in the `MarqoVectorStoreDriver` class:
 8. `create_index(self, name: str)`: Creates a new index in the Marqo client with the given name.
 9. `delete_index(self, name: str)`: Deletes an index in the Marqo client with the given name.
 10. `get_indexes(self)`: Returns a list of all indexes in the Marqo client.
-11. `upsert_vector(self, vector: list[float], vector_id: Optional[str] = None, namespace: Optional[str] = None, meta: Optional[dict] = None)`: Inserts a vector into the Marqo index. If a vector with the given vector ID already exists, it is updated; otherwise, a new vector is inserted. Currently, this function is not implemented and raises an Exception.
\ No newline at end of file
+11. `upsert_vector(self, vector: list[float], vector_id: Optional[str] = None, namespace: Optional[str] = None, meta: Optional[dict] = None)`: Inserts a vector into the Marqo index. If a vector with the given vector ID already exists, it is updated; otherwise, a new vector is inserted. Currently, this function is not implemented and raises an Exception.
+
+
+## MongoDBAtlasVectorStoreDriver
```
Comment (vasinov) [inline docs/griptape-framework/data/vector-stores.md:158]: Lower case "d" :)

File: docs/griptape-framework/data/vector-stores.md [hunk 2]
```diff
@@ -152,4 +152,62 @@ The following methods are available in the `MarqoVectorStoreDriver` class:
...
+import openai
+
+# Set the OpenAI API key
+openai.api_key_path = "../openai_api_key.txt"
+
+# Define the namespace
+namespace = "griptape-ai"
+
+# Initialize the vector store driver
+vector_store = MongoDBAtlasVectorStoreDriver(
```
Comment (vasinov) [inline docs/griptape-framework/data/vector-stores.md:179]: Lower case "d" :)

**Review by vasinov — CHANGES_REQUESTED** [review]

File: docs/griptape-framework/data/vector-stores.md (hunk 1 above)
Comment (trish11953) [reply docs/griptape-framework/data/vector-stores.md]: done!

File: docs/griptape-framework/data/vector-stores.md (hunk 2 above)
Comment (trish11953) [reply docs/griptape-framework/data/vector-stores.md]: Sorry, that was the older version i was doing yesterday, changed it with the new documentation!

**Review by vasinov — APPROVED** [review]

File: mkdocs.yml [hunk 3]
```diff
@@ -114,4 +114,5 @@ nav:
       - Query a Webpage: "examples/query-webpage.md"
     - Misc:
         - Count Tokens: "examples/count-tokens.md"
-        - Using Text Generation Web UI: "examples/using-text-generation-web-ui.md"
\ No newline at end of file
+        - Using Text Generation Web UI: "examples/using-text-generation-web-ui.md"
+        - Using Text Generation Web UI: "examples/dynamodb-conversation.md"
```
Comment (vasinov) [inline mkdocs.yml:118]: Can you update the title to something like "Storing Conversation Memory In DynamoDB"?

File: docs/griptape-framework/data/vector-stores.md [hunk 4]
```diff
@@ -152,4 +152,49 @@ The following methods are available in the `MarqoVectorStoreDriver` class:
 8. `create_index(self, name: str)`: Creates a new index in the Marqo client with the given name.
 9. `delete_index(self, name: str)`: Deletes an index in the Marqo client with the given name.
 10. `get_indexes(self)`: Returns a list of all indexes in the Marqo client.
-11. `upsert_vector(self, vector: list[float], vector_id: Optional[str] = None, namespace: Optional[str] = None, meta: Optional[dict] = None)`: Inserts a vector into the Marqo index. If a vector with the given vector ID already exists, it is updated; otherwise, a new vector is inserted. Currently, this function is not implemented and raises an Exception.
\ No newline at end of file
+11. `upsert_vector(self, vector: list[float], vector_id: Optional[str] = None, namespace: Optional[str] = None, meta: Optional[dict] = None)`: Inserts a vector into the Marqo index. If a vector with the given vector ID already exists, it is updated; otherwise, a new vector is inserted. Currently, this function is not implemented and raises an Exception.
+
+## MongoDbAtlasVectorStoreDriver
```
Comment (vasinov) [inline docs/griptape-framework/data/vector-stores.md:157]: Did you mean to commit those changes as well?

**Review by vasinov — CHANGES_REQUESTED** [review]

File: docs/griptape-framework/data/vector-stores.md (hunk 4 above)
Comment (trish11953) [reply docs/griptape-framework/data/vector-stores.md:157]: No changes were done to Marqodriver example. Only Mongodb was added.

File: mkdocs.yml (hunk 3 above)
Comment (trish11953) [reply mkdocs.yml:118]: Done!

**Review by vasinov — APPROVED** [review]

**Pull request merged**